examples/filter_file.py

```diff
@@ -0,0 +1,14 @@
+import os
+import sys
+import torchaudio
+
+if __name__ == '__main__':
```

Comment:
This is a nice code snippet, but demonstrating the `lowpass` would be better in [the audio tutorial](https://pytorch.org/tutorials/beginner/audio_preprocessing_tutorial.html) from [here](https://github.com/pytorch/tutorials/blob/master/beginner_source/audio_preprocessing_tutorial.py).

The good things from this example would be either if we want to offer some command line parsing, or if we want to use this as part of integrated testing. I'd open separate PRs though.

Thoughts?

Reply:
I agree.  I have removed this from latest update.